src/prefect/client/schemas/actions.py

```diff
@@ -617,3 +617,33 @@ class VariableUpdate(ActionBaseModel):
 
     # validators
     _validate_name_format = validator("name", allow_reuse=True)(validate_variable_name)
+
+
+@copy_model_fields
+class GlobalConcurrencyLimitCreate(ActionBaseModel):
+    """Data used by the Prefect REST API to create a global concurrency limit."""
+
+    name: str = FieldFrom(objects.GlobalConcurrencyLimit)
+    limit: int = FieldFrom(objects.GlobalConcurrencyLimit)
+    active: Optional[bool] = FieldFrom(objects.GlobalConcurrencyLimit)
+    active_slots: Optional[int] = FieldFrom(objects.GlobalConcurrencyLimit)
+    slot_decay_per_second: Optional[int] = FieldFrom(objects.GlobalConcurrencyLimit)
+
+
+@copy_model_fields
+class GlobalConcurrencyLimitUpdate(ActionBaseModel):
+    """Data used by the Prefect REST API to update a global concurrency limit."""
+
+    name: str = FieldFrom(objects.GlobalConcurrencyLimit)
+    limit: int = FieldFrom(objects.GlobalConcurrencyLimit)
```

Comment:
On update these both can be optional. The reason is that we'll be doing a `PATCH` which, in REST parlance, means we can send a partial object and expect that only the fields we send are going to be updated.

```suggestion
    name: Optional[str] = FieldFrom(objects.GlobalConcurrencyLimit)
    limit: Optional[int] = FieldFrom(objects.GlobalConcurrencyLimit)
```

Reply:
OK, i'll adapt that accordingly